config/packages/fos_elastica.yaml

```diff
@@ -226,4 +226,8 @@ fos_elastica:
                 publishedDate:
                     property_path: false
                     type: 'date'
-                    format: 'yyyy-MM-dd'                
+                    format: 'yyyy-MM-dd'
+                acceptedDate:
```

Comment:
If there is a better way to treat both accepted date for datasets and published date on info products indexes the same, that would be a better approach than faking a redundant field.